docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -52,6 +53,14 @@ \subsection{Inputs and Outputs}
 
 \subsection{Stakeholders}
 
+\begin{itemize}
+    \item Students of McMaster University
+    \item Single income households
+    \item Potential users looking to save money on groceries
```

Comment:
this feels too vague... anybody can be a "potential user"

Reply:
Perhaps instead of including it in the list, you could add it as a separate sentence? For example:

```
- stakeholder 1
- stakeholder 2
- ...

"In general, this application could be used by anyone looking to save money on groceries"
```

Alternatively, you could just remove it.

Reply:
Removed the line.